packages/bcp/src/cosmwasmcodec.ts

```diff
@@ -15,18 +22,18 @@ import {
   UnsignedTransaction,
 } from "@iov/bcp";
 
-import { CosmosBech32Prefix, isValidAddress, pubkeyToAddress } from "./address";
+import { pubkeyToAddress } from "./address";
 import { Caip5 } from "./caip5";
 import { parseTx } from "./decode";
 import { buildSignedTx, buildUnsignedTx } from "./encode";
 import { nonceToAccountNumber, nonceToSequence, TokenInfos } from "./types";
 
 export class CosmWasmCodec implements TxCodec {
-  private readonly prefix: CosmosBech32Prefix;
+  private readonly addressPrefix: CosmosAddressBech32Prefix;
   private readonly tokens: TokenInfos;
 
-  public constructor(prefix: CosmosBech32Prefix, tokens: TokenInfos) {
-    this.prefix = prefix;
+  public constructor(prefix: CosmosAddressBech32Prefix, tokens: TokenInfos) {
```

Comment:
I wonder about not just making this a string.
It is true for the hub, but most zones make their own prefixes. Eg. Xrn:... for regen